diff --git a/node_modules/eslint-plugin-no-secrets/index.js b/node_modules/eslint-plugin-no-secrets/index.js
index c6aa6d6..8306ff7 100644
--- a/node_modules/eslint-plugin-no-secrets/index.js
+++ b/node_modules/eslint-plugin-no-secrets/index.js
@@ -125,15 +125,5 @@ module.exports = {
         };
       }
     }
-  },
-  processors: {
-    ".json": {
-      preprocess(text) {
-        return [`var json = ${text}`];
-      },
-      postprocess(messages, fileName) {
-        return messages[0];
-      }
-    }
   }
 };
